server/pbench/bin/pbench-rsync-satellite

```diff
@@ -119,6 +114,12 @@ for host in $hosts ;do
         echo "FAILED:    rsync -av $remotehost:$remotearchive/$host/ ."
         let failures=failures+1
     fi
+    # move the symlink from TO-SYNC directory to SYNCED on the remote server.
```

Comment:
I discussed these two lines of code with Nick this afternoon.  We believe the new line 121 will end up with a race condition with a new `move-results` happening at the same time.  To void that, let's merge these to steps, line 118 and line 121 in this commit, with the archive removal further down at line 137.

Then we can have that script operate line-by-line on the tarballs to remove, where it:

 1. Moves the link from `TO-SYNC` to `SYNCED`
 2. Then removes the `.prefix` entry for that tar ball
 3. Finally removing the archive itself